frontend/pages/index.tsx

```diff
@@ -11,15 +11,28 @@ export const getServerSideProps: GetServerSideProps = async context => {
     const session = await getSession(context);
     const role = session?.user.Role;
     let data: unknown[] = [];
+    let userId = session?.user.AccountId;
+    let statusOfTheDay = [];
+    const today = new Date().toISOString().slice(0, 10);
+    let statusFilled = true;
+
+    if (role === USER_ROLES.patient) {
+        let response = await fetch(serverURL + "/status/getAllStatusChart/" + userId);
+        data = await response.json();
+
+        // status filled for the day?
+        response = await fetch(serverURL + "/status/getAllStatus/" + userId);
+        statusOfTheDay = await response.json();
+
+        statusFilled = statusOfTheDay[0].StatusTime.substring(0, 10) === today;
+    }
```

Comment:
you could combine all these promises with `Promise.all([promise1, promise2, promise3])`